test/unit/test_preprocessing.py

```diff
@@ -44,6 +46,17 @@ def _make_image_seq(imgs):
     imgs = [ModuleImage(img, EL_IMAGE, None) for img in imgs]
     return ModuleImageSequence(imgs, True, False)
 
+def _prepare_stitching_test_img():
+    image = data.datasets.poly10x6(1)._images[0]._data
```

Comment:
Please do not access the private properties `_images` and `_data`. They are not part of the API. You can access the data of the first image of an `ImageSequence` as `sequence[0].data`

Reply:
In the same file 38th row(version without my code) or 40th row(version with my code): 
ds["{:d}A".format(ex_all[1])][0]._data, EL_IMAGE, path=Path() / "test.png"  
there is also a        ._data 
 I hope this is helpful for you